frontend/app_flowy/packages/appflowy_backend/lib/ffi.dart

```diff
@@ -133,3 +133,15 @@ typedef _store_dart_post_cobject_C = Void Function(
 typedef _store_dart_post_cobject_Dart = void Function(
   Pointer<NativeFunction<Int8 Function(Int64, Pointer<Dart_CObject>)>> ptr,
 );
+
+void send_log_to_backend(
```

Comment:
I think we can rename the function to `log` and then just calling `ffi.log("xx")`.